activitysim/abm/models/vehicle_type_choice.py

```diff
@@ -231,7 +231,9 @@ def construct_model_alternatives(
         else:
             # eliminate alternatives if no vehicle type data
             alts_wide = alts_wide[alts_wide._merge != "left_only"]
-        alts_wide.drop(columns="_merge", inplace=True)
+        alts_wide.drop(
+            columns="_merge", inplace=True
+        )  # if this happens, alt_wide length will not match alt_long
```

Comment:
Considering the comment, is this then a bug that needs to be addressed?

Reply:
Opened issue: https://github.com/ActivitySim/activitysim/issues/789

Reply:
I moved this comment up two lines.